esmvaltool/cmorizers/obs/cmorize_obs_ncepslp.py

```diff
@@ -0,0 +1,67 @@
+"""ESMValTool CMORizer for NCEP data.
+
+Tier
+    Tier 2: open dataset.
+
+Source
+    https://www.psl.noaa.gov/data/gridded/data.ncep.reanalysis.derived.html
+
+Last access
+    20200520
+
+Download and processing instructions
+    TODO
```

Comment:
@soufianekar : Could you add some info here how to download the data

Reply:
ftp://ftp.cdc.noaa.gov/Datasets/ncep.reanalysis.derived/surface/slp.mon.mean.nc